src/main/java/org/gaul/s3proxy/S3ProxyHandlerJetty.java

```diff
@@ -50,6 +54,13 @@
                 ignoreUnknownHeaders, corsAllowAll, servicePath);
     }
 
+    private void sendS3Exception(HttpServletRequest request,
```

Comment:
Sorry I missed the access to `handler` -- agree that this looked better before my errant suggestion.